std/codec.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package cosmos_sdk.codec.std.v1;
```

Comment:
I presume this file will be deleted once we update all the modules?

Reply:
Yep